internal/controller/dnsrecord_controller.go

```diff
@@ -470,12 +478,6 @@ func (r *DNSRecordReconciler) updateStatusAndRequeue(ctx context.Context, previo
 	}
 	logger.V(1).Info(fmt.Sprintf("Requeue in %s", requeueTime.String()))
 
-	var gauge float64
-	if meta.IsStatusConditionTrue(current.Status.Conditions, string(v1alpha1.ConditionTypeReady)) {
-		gauge = 1
-	}
-	metrics.RecordReady.WithLabelValues(current.Name, current.Namespace, current.Spec.RootHost, strconv.FormatBool(current.IsDelegating())).Set(gauge)
-
```

Comment:
@maksymvavilov Currently the remote dnsrecord controller has a duplicate of most things in here (short term thing), including `updateStatusAndRequeue`, https://github.com/Kuadrant/dns-operator/pull/573/files#diff-d5d1c2653accecb155c83e4b05ba794e0b7ec17db6abe72e430522c4c06bf230R367-R372, you'll need to remove these from it's version also.